### ISSUE TITLE
DART Runner is changing filenames

### ISSUE THREAD

**Issue by bonniegee** [APTrust/dart-runner#6]
When running DART Runner, filenames are unexpectedly changing. Filenames are overwritten with a number.

The command I'm running: `./dart-runner --workflow=/data/dart_runner_files/digitization_workflow.json --output-dir=/mnt/IIIF_Testing_Area/tmp`

The JSON job params:
```
{'packageName': '853dbf149078419e87a69856394cbf6f.tar', 'files': ['/mnt/IIIF_Testing_Area/tmp/853dbf149078419e87a69856394cbf6f/853dbf149078419e87a69856394cbf6f_001.tif', '/mnt/IIIF_Testing_Area/tmp/853dbf149078419e87a69856394cbf6f/853dbf149078419e87a69856394cbf6f_002.tif', '/mnt/IIIF_Testing_Area/tmp/853dbf149078419e87a69856394cbf6f/853dbf149078419e87a69856394cbf6f_003.tif', '/mnt/IIIF_Testing_Area/tmp/853dbf149078419e87a69856394cbf6f/853dbf149078419e87a69856394cbf6f_004.tif', '/mnt/IIIF_Testing_Area/tmp/853dbf149078419e87a69856394cbf6f/853dbf149078419e87a69856394cbf6f_005.tif', '/mnt/IIIF_Testing_Area/tmp/853dbf149078419e87a69856394cbf6f/853dbf149078419e87a69856394cbf6f_006.tif'], 'tags': [{'tagFile': 'bag-info.txt', 'tagName': 'ArchivesSpace-URI', 'value': '/repositories/2/archival_objects/32987'}, {'tagFile': 'bag-info.txt', 'tagName': 'Start-Date', 'value': '1912-01-01'}, {'tagFile': 'bag-info.txt', 'tagName': 'End-Date', 'value': '1912-12-31'}, {'tagFile': 'bag-info.txt', 'tagName': 'Origin', 'userValue': 'digitization'}, {'tagFile': 'bag-info.txt', 'tagName': 'Rights-ID', 'value': '1'}, {'tagFile': 'bag-info.txt', 'tagName': 'Rights-ID', 'value': '6'}]}
```
The output:
```
>>> child.communicate(job_json_input)
(b'{"jobName":"853dbf149078419e87a69856394cbf6f.tar","payloadByteCount":139918737,"payloadFileCount":6,"succeeded":true,"packageResult":{"attempt":1,"completed":"2022-06-03T15:41:31.211232989-04:00","errors":{},"fileMtime":"2022-06-03T15:41:31.2105843-04:00","filepath":"/mnt/IIIF_Testing_Area/tmp/853dbf149078419e87a69856394cbf6f.tar","filesize":139928064,"info":"","operation":"package","provider":"Bagger - ","remoteChecksum":"","remoteURL":"","started":"2022-06-03T15:41:01.157105313-04:00","warning":""},"validationResult":{"attempt":1,"completed":"2022-06-03T15:41:31.214479732-04:00","errors":{},"fileMtime":"2022-06-03T15:41:31.2105843-04:00","filepath":"/mnt/IIIF_Testing_Area/tmp/853dbf149078419e87a69856394cbf6f.tar","filesize":139928064,"info":"","operation":"validation","provider":"Validator - ","remoteChecksum":"","remoteURL":"","started":"2022-06-03T15:41:31.211265138-04:00","warning":""},"uploadResults":[{"attempt":1,"completed":"2022-06-03T15:42:25.7519241-04:00","errors":{},"fileMtime":"2022-06-03T15:41:31.2437404-04:00","filepath":"/mnt/IIIF_Testing_Area/tmp/853dbf149078419e87a69856394cbf6f.tar","filesize":139928064,"info":"Output file at /mnt/IIIF_Testing_Area/tmp/853dbf149078419e87a69856394cbf6f.tar was deleted at 2022-06-03T15:42:25-04:00","operation":"upload","provider":"Uploader - ","remoteChecksum":"cc3a81acebe97b6f46b06a24011da36f-9","remoteURL":"s3://s3.amazonaws.com/rac-zorya-staging/853dbf149078419e87a69856394cbf6f.tar","started":"2022-06-03T15:41:31.214481005-04:00","warning":""}],"validationErrors":null}\n', None)
```
The list of files from the manifest for the bag that DART Runner creates:
```
  data/1.tif
  data/2.tif
  data/3.tif
  data/4.tif
  data/5.tif
  data/6.tif
```

**Comment by diamondap** [comment]
This should be fixed in v0.93-beta. See https://aptrust.github.io/dart-docs/users/dart_runner/#downloads for download links.

**Comment by bonniegee** [comment]
Thank you, @diamondap !
